tox.ini

```
@@ -1,7 +1,7 @@
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = lint, unit, integration
+envlist = lint, static, unit, integration
```

```suggestion
envlist = lint, static, unit, integration, prettify
```

Should `prettify` be included here?

---

envlist is what runs if you execute tox without arguments. so prettify should probably be excluded?

---

Ah ofc, my bad.